examples/filter2/main.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	logging "github.com/ipfs/go-log"
+	"github.com/status-im/go-waku/waku/v2/node"
+	"github.com/status-im/go-waku/waku/v2/protocol/pb"
+)
+
+var log = logging.Logger("filter2")
+
+const pubSubTopic = ""
```

Comment:
Should this be empty?

Reply:
Just used `relay.DefaultWakuTopic` here and in calls to `SubscribeFilter()` and `Subscribe()`